src/widget/friendlistlayout.cpp

```diff
@@ -72,10 +72,11 @@ int FriendListLayout::indexOfFriendWidget(GenericChatItemWidget* widget, bool on
 {
     if (online)
         return friendOnlineLayout.indexOfSortedWidget(widget);
+
     return friendOfflineLayout.indexOfSortedWidget(widget);
 }
 
-void FriendListLayout::moveFriendWidgets(FriendListWidget* listWidget)
+void FriendListLayout::moveFriendWidgets(FriendListWidget* listWidget) const
```

Comment:
The whole purpose of this function is to move elements of the layout around, it's not const at all!
Semantically, it makes no sense. The goal of the function is to modify things, it's not a const function.